include/flatbuffers/flatbuffers.h

```diff
@@ -803,7 +803,12 @@ class vector_downward {
   uint8_t *data_at(size_t offset) const { return buf_ + reserved_ - offset; }
 
   void push(const uint8_t *bytes, size_t num) {
-    memcpy(make_space(num), bytes, num);
+    // CreateVector above, can pass through a nullptr for bytes
```

Comment:
If so, lets make the fix in CreateVector, not here. This is performance sensitive code, so I would like as few if-thens (or as far from the core) as possible.

Reply:
OK, are there benchmarks as part of the project I can run to see if there will be impact from my changes?

Reply:
No there aren't. There are some super old benchmarks in a branch that are win32 only.

Regardless, if you can pull an if-then further away from a hot function, why not do it? especially when all other callers of this function already comply with `memcpy`'s rules?

Reply:
I think its fine, just wondering if I can assess the impact further upstream.  I will update the PR in the next day or two.  Thank you for the feedback.